### PR TITLE
HBASE-25424 Find a way to config OpenTelemetry tracing without direct…

### PR DESCRIPTION
**Pull request by Apache9**
…ly depending on opentelemetry-sdk

### CODE REVIEW

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 30s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ HBASE-22120 Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 30s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 14s |  Maven dependency ordering for patch  |
| -0 :warning: |  shellcheck  |   0m  2s |  The patch generated 3 new + 34 unchanged - 0 fixed = 37 total (was 34)  |
| -0 :warning: |  whitespace  |   0m  0s |  The patch has 2 line(s) that end in whitespace. Use git apply --whitespace=fix <<patch_file>>. Refer https://git-scm.com/docs/git-apply  |
| +1 :green_heart: |  xml  |   0m  4s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  hadoopcheck  |  17m 32s |  Patch does not cause any errors with Hadoop 3.1.2 3.2.1 3.3.0.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 29s |  The patch does not generate ASF License warnings.  |
|  |   |  27m 18s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/1/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2808 |
| Optional Tests | dupname asflicense shellcheck shelldocs hadoopcheck xml |
| uname | Linux 58b822d742a9 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | HBASE-22120 / bd745da477 |
| shellcheck | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/1/artifact/yetus-general-check/output/diff-patch-shellcheck.txt |
| whitespace | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/1/artifact/yetus-general-check/output/whitespace-eol.txt |
| Max. process+thread count | 76 (vs. ulimit of 30000) |
| modules | C: . hbase-assembly U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/1/console |
| versions | git=2.17.1 maven=3.6.3 shellcheck=0.4.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 28s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ HBASE-22120 Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 31s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m 12s |  HBASE-22120 passed  |
| +1 :green_heart: |  compile  |   2m 48s |  HBASE-22120 passed  |
| +1 :green_heart: |  shadedjars  |   7m  9s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   3m  1s |  HBASE-22120 passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 22s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   4m  2s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 56s |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 56s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   7m  0s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 58s |  the patch passed  |
||| _ Other Tests _ |
| -1 :x: |  unit  | 144m  1s |  root in the patch failed.  |
|  |   | 182m  6s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/1/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2808 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 4a0a4da538ad 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | HBASE-22120 / bd745da477 |
| Default Java | AdoptOpenJDK-11.0.6+10 |
| unit | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/1/artifact/yetus-jdk11-hadoop3-check/output/patch-unit-root.txt |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/1/testReport/ |
| Max. process+thread count | 4147 (vs. ulimit of 30000) |
| modules | C: hbase-assembly . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/1/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   9m 22s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  1s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ HBASE-22120 Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 30s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 15s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  shellcheck  |   0m  2s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  5s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  hadoopcheck  |  20m 36s |  Patch does not cause any errors with Hadoop 3.1.2 3.2.1 3.3.0.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 28s |  The patch does not generate ASF License warnings.  |
|  |   |  39m 12s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.40 ServerAPI=1.40 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/2/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2808 |
| Optional Tests | dupname asflicense shellcheck shelldocs hadoopcheck xml |
| uname | Linux 997d5111a4ed 4.15.0-101-generic #102-Ubuntu SMP Mon May 11 10:07:26 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | HBASE-22120 / bd745da477 |
| Max. process+thread count | 63 (vs. ulimit of 30000) |
| modules | C: . hbase-assembly U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/2/console |
| versions | git=2.17.1 maven=3.6.3 shellcheck=0.4.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 14s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  4s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ HBASE-22120 Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 34s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m 19s |  HBASE-22120 passed  |
| +1 :green_heart: |  compile  |   2m 50s |  HBASE-22120 passed  |
| +1 :green_heart: |  shadedjars  |   7m  3s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 57s |  HBASE-22120 passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 22s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 59s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 51s |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 51s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   6m 58s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 56s |  the patch passed  |
||| _ Other Tests _ |
| -1 :x: |  unit  | 149m 47s |  root in the patch failed.  |
|  |   | 188m 28s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/2/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2808 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 8055092030e8 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | HBASE-22120 / bd745da477 |
| Default Java | AdoptOpenJDK-11.0.6+10 |
| unit | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/2/artifact/yetus-jdk11-hadoop3-check/output/patch-unit-root.txt |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/2/testReport/ |
| Max. process+thread count | 3857 (vs. ulimit of 30000) |
| modules | C: hbase-assembly . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/2/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

File: conf/hbase-env.sh [hunk 1]
```diff
@@ -142,3 +142,7 @@
 # Override text processing tools for use by these launch scripts.
 # export GREP="${GREP-grep}"
 # export SED="${SED-sed}"
+
+# Uncomment to enable trace, you can change the options to use other exporters such as jaeger or
+# zipkin
+# export HBASE_TRACE_OPTS="-Dotel.config.sampler.probability=0.1 -Dotel.exporter=logging"
```
Comment (infraio) [inline conf/hbase-env.sh:148]: Can we add a url to show that how to config otel.exporter?

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 28s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ HBASE-22120 Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 35s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 13s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  shellcheck  |   0m  2s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  3s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  hadoopcheck  |  17m 48s |  Patch does not cause any errors with Hadoop 3.1.2 3.2.1 3.3.0.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 30s |  The patch does not generate ASF License warnings.  |
|  |   |  26m 30s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/3/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2808 |
| Optional Tests | dupname asflicense shellcheck shelldocs hadoopcheck xml |
| uname | Linux 0023de2a58ee 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | HBASE-22120 / bd745da477 |
| Max. process+thread count | 76 (vs. ulimit of 30000) |
| modules | C: . hbase-assembly U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/3/console |
| versions | git=2.17.1 maven=3.6.3 shellcheck=0.4.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Review by infraio — APPROVED** [review]
+1

**Comment by Apache-HBase** [comment]
:confetti_ball: **+1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   0m 29s |  Docker mode activated.  |
||| _ Prechecks _ |
| +1 :green_heart: |  dupname  |   0m  0s |  No case conflicting files found.  |
| +0 :ok: |  shelldocs  |   0m  0s |  Shelldocs was not available.  |
| +1 :green_heart: |  @author  |   0m  0s |  The patch does not contain any @author tags.  |
||| _ HBASE-22120 Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 19s |  Maven dependency ordering for branch  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 19s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  shellcheck  |   0m  1s |  There were no new shellcheck issues.  |
| +1 :green_heart: |  whitespace  |   0m  0s |  The patch has no whitespace issues.  |
| +1 :green_heart: |  xml  |   0m  4s |  The patch has no ill-formed XML file.  |
| +1 :green_heart: |  hadoopcheck  |  17m  8s |  Patch does not cause any errors with Hadoop 3.1.2 3.2.1 3.3.0.  |
||| _ Other Tests _ |
| +1 :green_heart: |  asflicense  |   0m 27s |  The patch does not generate ASF License warnings.  |
|  |   |  25m 40s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/4/artifact/yetus-general-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2808 |
| Optional Tests | dupname asflicense shellcheck shelldocs hadoopcheck xml |
| uname | Linux def44d78dba3 4.15.0-60-generic #67-Ubuntu SMP Thu Aug 22 16:55:30 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | HBASE-22120 / bd745da477 |
| Max. process+thread count | 76 (vs. ulimit of 30000) |
| modules | C: hbase-assembly . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/4/console |
| versions | git=2.17.1 maven=3.6.3 shellcheck=0.4.6 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m  1s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ HBASE-22120 Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 19s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   4m  0s |  HBASE-22120 passed  |
| +1 :green_heart: |  compile  |   2m 51s |  HBASE-22120 passed  |
| +1 :green_heart: |  shadedjars  |   7m  0s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 57s |  HBASE-22120 passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 19s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   4m  3s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 50s |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 50s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   7m  0s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 53s |  the patch passed  |
||| _ Other Tests _ |
| -1 :x: |  unit  | 150m 59s |  root in the patch failed.  |
|  |   | 188m 24s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/4/artifact/yetus-jdk11-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2808 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 552a83048cbd 4.15.0-112-generic #113-Ubuntu SMP Thu Jul 9 23:41:39 UTC 2020 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | HBASE-22120 / bd745da477 |
| Default Java | AdoptOpenJDK-11.0.6+10 |
| unit | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/4/artifact/yetus-jdk11-hadoop3-check/output/patch-unit-root.txt |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/4/testReport/ |
| Max. process+thread count | 3890 (vs. ulimit of 30000) |
| modules | C: hbase-assembly . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/4/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Comment by Apache-HBase** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| +0 :ok: |  reexec  |   1m 13s |  Docker mode activated.  |
| -0 :warning: |  yetus  |   0m  3s |  Unprocessed flag(s): --brief-report-file --spotbugs-strict-precheck --whitespace-eol-ignore-list --whitespace-tabs-ignore-list --quick-hadoopcheck  |
||| _ Prechecks _ |
||| _ HBASE-22120 Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 17s |  Maven dependency ordering for branch  |
| +1 :green_heart: |  mvninstall  |   3m 25s |  HBASE-22120 passed  |
| +1 :green_heart: |  compile  |   2m 26s |  HBASE-22120 passed  |
| +1 :green_heart: |  shadedjars  |   6m 58s |  branch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 21s |  HBASE-22120 passed  |
||| _ Patch Compile Tests _ |
| +0 :ok: |  mvndep  |   0m 19s |  Maven dependency ordering for patch  |
| +1 :green_heart: |  mvninstall  |   3m 26s |  the patch passed  |
| +1 :green_heart: |  compile  |   2m 19s |  the patch passed  |
| +1 :green_heart: |  javac  |   2m 19s |  the patch passed  |
| +1 :green_heart: |  shadedjars  |   6m 50s |  patch has no errors when building our shaded downstream artifacts.  |
| +1 :green_heart: |  javadoc  |   2m 19s |  the patch passed  |
||| _ Other Tests _ |
| -1 :x: |  unit  | 317m 32s |  root in the patch failed.  |
|  |   | 351m 56s |   |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | ClientAPI=1.41 ServerAPI=1.41 base: https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/4/artifact/yetus-jdk8-hadoop3-check/output/Dockerfile |
| GITHUB PR | https://github.com/apache/hbase/pull/2808 |
| Optional Tests | javac javadoc unit shadedjars compile |
| uname | Linux 5f20a1483d07 4.15.0-65-generic #74-Ubuntu SMP Tue Sep 17 17:06:04 UTC 2019 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | dev-support/hbase-personality.sh |
| git revision | HBASE-22120 / bd745da477 |
| Default Java | AdoptOpenJDK-1.8.0_232-b09 |
| unit | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/4/artifact/yetus-jdk8-hadoop3-check/output/patch-unit-root.txt |
|  Test Results | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/4/testReport/ |
| Max. process+thread count | 4278 (vs. ulimit of 30000) |
| modules | C: hbase-assembly . U: . |
| Console output | https://ci-hadoop.apache.org/job/HBase/job/HBase-PreCommit-GitHub-PR/job/PR-2808/4/console |
| versions | git=2.17.1 maven=3.6.3 |
| Powered by | Apache Yetus 0.12.0 https://yetus.apache.org |


This message was automatically generated.

**Pull request merged**